### PR TITLE
refactor: move getPreset into lib/config/preset/

### PR DESCRIPTION
**Pull request by rarkins**

### CODE REVIEW

File: lib/config/presets/npm.ts [hunk 1]
```diff
@@ -1,11 +1,10 @@
 import { logger } from '../../logger';
-import { getDependency } from './get';
-import { Preset } from '../common';
+import { getDependency } from '../../datasource/npm/get';
 
 export async function getPreset(
   pkgName: string,
   presetName = 'default'
-): Promise<Preset> {
+): Promise<any> {
```
Comment (viceice) [inline lib/config/presets/npm.ts]: Why this?

Comment (rarkins) [reply lib/config/presets/npm.ts]: `Preset` was just an alias for `any` anyway, so I couldn't be bothered creating a new `custom.ts` just for that.

Comment (viceice) [reply lib/config/presets/npm.ts]: ok, but we should add a type for it later.

Comment (viceice) [reply lib/config/presets/npm.ts]: I want to ge rid of almost any `any` in future

Comment (rarkins) [reply lib/config/presets/npm.ts]: Fixing it

**Review by viceice — COMMENTED** [review]
👀 LGTM

**Pull request merged**